Formula/pastebinit.rb

```diff
@@ -19,6 +11,7 @@ def install
     inreplace "pastebinit", "/usr/bin/python3", Formula["python3"].opt_bin + "python3"
     inreplace "pastebinit", "/usr/local/etc/pastebin.d", etc + "pastebin.d"
     system "docbook2man", "pastebinit.xml"
+    rm_r etc/"pastebin.d"
```

Comment:
In v1.4.1, the API key is wrong for http://pastebin.com (in `pastebin.d/pastebin.com.conf`). That's why I need to replace the files in `pastebin.d`.